docs/UserGuide.adoc

```diff
@@ -448,74 +448,94 @@ Represents the Facebook account with the username `johnd10`.
 * `ig damyth` +
 represents the Instagram account with the username `damyth`.
 
+[[facebook-features]]
 ==== Facebook Integrations
 
-===== Connecting to Facebook [Since v1.2]: `facebook connect`
+// tag::facebookconnect[]
+===== Connecting to Facebook [Since v1.2]: `facebookconnect`
 
 Connects to your Facebook account. +
-Format: `facebook connect`
+Format: `facebookconnect`
```

Comment:
Can we change "format" to "Command Format" for consistency? 

Also, please include colors for the command format to make it clearer which is the command word, the option, and the parameters!